frontend/src/views/DataForms/BusinessTripForm.tsx

```diff
@@ -193,6 +193,7 @@ export function BusinessTripForm(
       <MenuItem value={2020}>2020</MenuItem>
       <MenuItem value={2021}>2021</MenuItem>
       <MenuItem value={2022}>2022</MenuItem>
+      <MenuItem value={2023}>2023</MenuItem>
```

Comment:
I know we have an issue to setup a datepicker but for now we can move all these year menu items to one place and reuse that. This is too much duplication

Reply:
extracted them into constants until we can use datepickers